apps/studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx

```diff
@@ -45,19 +45,22 @@ const ComputeInstanceSidePanel = () => {
   const { ref: projectRef } = useParams()
   const { project: selectedProject } = useProjectContext()
   const organization = useSelectedOrganization()
-
   const computeSizeChangesDisabled = useFlag('disableComputeSizeChanges')
+  const diskAndComputeForm = useFlag('diskAndComputeForm')
   const [showConfirmationModal, setShowConfirmationModal] = useState(false)
-
   const canUpdateCompute = useCheckPermissions(
     PermissionAction.BILLING_WRITE,
     'stripe.subscriptions'
   )
-
   const isProjectActive = useIsProjectActive()
-
   const { panel, setPanel, closePanel } = useAddonsPagePanel()
+
   const visible = panel === 'computeInstance'
+  if (visible) {
```

Comment:
can we put this in a useEffect? so it only runs once

Reply:
done